homework-02

yetoneya opened the pull request.

yetoneya committed:
Create .travis.yml

yetoneya committed:
Update .travis.yml

yetoneya committed:
Update .travis.yml

yetoneya committed:
Update .travis.yml

yetoneya committed:
Update README.md

yetoneya committed:
Update README.md

yetoneya committed:
Update README.md

yetoneya committed:
Update README.md

mrgreyves reviewed (changes requested):
Здравствуйте! Проверил ваше дз. Все домашние задания нужно выполнять в репозиториях созданных для вас. Так же если это дз play-travis то в репозитории не хватает файлов. ДЗ возвращаю на доработку.

yetoneya committed:
Update README.md

yetoneya committed:
travis

yetoneya committed:
Update README.md

Update README.md

Update README.md

Update .travis.yml

Update .travis.yml

Update .travis.yml

travis

travis

yetoneya commented:
> Здравствуйте! Проверил ваше дз. Все домашние задания нужно выполнять в репозиториях созданных для вас. Так же если это дз play-travis то в репозитории не хватает файлов. ДЗ возвращаю на доработку.

 Permission to Otus-DevOps-2020-11/yetoneya_infra.git denied to yetoneya? Недостаточно прав.
Файлы добавила. Сразу не добавила, ппотому что не знала, зачем они

а почему travis в списке организаций не видит Otus-DevOps-2020-11?

mrgreyves commented on .travis.yml:
```diff
@@ -0,0 +1,9 @@
+sudo: required
+language: bash
+before_install:
+- curl https://raw.githubusercontent.com/express42/otus-homeworks/2020-11/run.sh | bashdist: trusty
```
Ошибка в строке

mrgreyves reviewed (changes requested):
Здравствуйте! Еще раз проверил ваше дз. Разрешения на пуш не было потому что вы не приняли приглашение. Добавил вам права руками. Так же нужно добавить описание к пр и переделать название ветки из которой делается пр. ДЗ возвращаю на доработку.
